mint/fakeAPIData.ts

```diff
@@ -393,5 +473,49 @@ export const settings = [
             {value: 0.9, label: "90%"},
             {value: 1, label: "100%"}
         ]
+    },
+    {
+        "id": "population",
```

Comment:
So when this is integrated into MINT, it will use the combined baseline and intervention settings to allow the calculation of the formula? (Population is a baseline setting)

Reply:
Yeah exactly: https://github.com/mrc-ide/mint/pull/53/files#diff-c4e14ba104c66f96849bc8b1bacef1800f00366f85dbb557336f03eea6160bb2R46